app/views/questions/new.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <h3>New Question</h2>
+    <div class="errors"><%= erb :'partials/_save_errors', locals: { object: @question } %></div>
+    <%= erb :'partials/_question_form', locals: {path: @path, method: @method, question: @question} %>
```

Comment:
Ah, so here are where those variables I commented on earlier are needed. Nice use of a partial! I still think it's better to just put the path/method right here, though.